ruby/hashes.rb

```diff
@@ -80,9 +84,42 @@
 
 }
 
-#Print hash out to screen
+#Print hash to screen for user to review; uses .each iteration method to print has keys & values in user-friendly readable statements
+puts "Please review and confirm your responses below:"
+
+new_client_info.each do |detail, value|
+    puts "Client's #{detail.to_s.tr('_', ' ')}: #{value}"
+end
+
 
 
 
 #Provide user chance to update or correct values, skipping is user enters 'none'
-#If user enters a key they want to change, prompt for new value and update
\ No newline at end of file
+#If user enters a key they want to change, prompt for new value and update
+puts "If there is a response you'd like to change, enter which value you want to update (name, number_of_children, approximate_budget etc.),
+then enter the updated information when prompted. When you are finished, or if you have no changes to make, type 'none' to skip this step"
+
+loop do 
```

Comment:
`loop..do` does not describe the type of loop you are constructing. What you are tying to express is "Run the loop until the user enters 'none'" This would lend itself better to a while or until loop.